app/parse.py

```diff
@@ -15,8 +33,96 @@ class Product:
     num_of_reviews: int
 
 
+PRODUCT_FIELDS = [field.name for field in fields(Product)]
+
+
+def init_driver(headless: bool = True) -> webdriver.Chrome:
+    options = Options()
+    options.headless = headless
+    return webdriver.Chrome(options=options)
+
+
+def accept_cookies(driver: webdriver.Chrome) -> None:
+    try:
+        cookie = WebDriverWait(driver, 3).until(
+            ec.presence_of_element_located((By.CLASS_NAME, "acceptCookies"))
+        )
+        cookie.click()
+    except (ElementClickInterceptedException, TimeoutException) as e:
+        print(f"Error accepting cookies: {e}")
+
+
+def click_load_more(driver: webdriver.Chrome) -> None:
+    try:
+        while True:
+            button = WebDriverWait(driver, 1).until(
+                ec.presence_of_element_located((By.CLASS_NAME, "ecomerce-items-scroll-more"))
+            )
+            if button.is_displayed():
+                button.click()
+                time.sleep(0.2)
+            else:
+                break
+    except (TimeoutException, ElementClickInterceptedException):
```

Comment:
The `click_load_more` function assumes that the 'Load More' button is always present and clickable. If the button is not present or not clickable, the function will print a message and exit the loop. Consider adding more robust error handling or checks to ensure the button's presence and state.